src/components/Notifications/DataAlerts/DataAlerts.js

```diff
@@ -215,8 +197,9 @@ class DataAlerts extends React.Component {
 
     return (
       <div className='data-alerts-list-container'>
-        {type === 'custom' &&
-          this.renderNotificationGroupTitle('Custom Data Alerts', 'View and manage your custom Data Alerts', true)}
+        {type === 'custom' && _get(list, 'length')
```

Comment:
We could probably just use list?.length instead of lodash here

Reply:
yep, for sure!